client/components/pages/configuration/users/UsersTable.vue

```diff
@@ -60,6 +60,16 @@ function onPage(event: DataTablePageEvent) {
 function onRowContextMenu(event: DataTableRowContextMenuEvent) {
   contextMenu.value?.show(event.originalEvent)
 }
+
+useEventBus(WebsocketPacketReceivedEvent).on(({ payload }) => {
+  switch (payload.type) {
+    case 'file-uploaded':
```

Comment:
'file-uploaded' type does not exist as a possible payload. See HederaWebsocketPayload type